modules/KIWIImage.pm

```diff
@@ -4125,9 +4125,17 @@ sub extractLinux {
     my $xml       = $this->{xml};
     my $vconf     = $xml -> getVMachineConfig();
     my $xendomain;
-    my $kernname = "vmlinuz";
-    if (-f "$imageTree/boot/vmlinux") {
-        $kernname = "vmlinux";
+    my $kernname;
+    foreach (
+        "image",   # s390x
+        "vmlinux", # ppc*
+        "Image",   # aarch64
+        "uImage",  # armv* (legacy)
+        "zImage",  # armv*
+        "vmlinuz", # x86*
+        ) {
```

Comment:
ah, I see, I wasn't calling suseStripKernel, so thats why. how about the following patch instead (untested)?